rpc/rpc-wrappers.go

```diff
@@ -1051,8 +1080,8 @@ type StorageAuthorityClient struct {
 }
 
 // NewStorageAuthorityClient constructs an RPC client
-func NewStorageAuthorityClient(client RPCClient) (sac StorageAuthorityClient, err error) {
-	sac = StorageAuthorityClient{rpc: client}
+func NewStorageAuthorityClient(client RPCClient) (cac StorageAuthorityClient, err error) {
```

Comment:
This looks like a spurious diff, and inaccurate since `cac` would usually refer to a CA client.